remote_error_monitor.c

```diff
@@ -157,7 +157,7 @@ static void remote_error_monitor_error_callback(int type, const char *error_file
   smart_str trace_str = {0};
   //rem_append_backtrace(&trace_str);
   smart_str_0(&trace_str);
-  remote_error_monitor_process(REMOTE_ERROR_MONITOR_ERROR, error_filename, error_lineno, args, trace_str.s ? ZSTR_VAL(trace_str.s) : "");
+  remote_error_monitor_process(type, error_filename, error_lineno, args, trace_str.s ? ZSTR_VAL(trace_str.s) : "");
```

Comment:
If we do not want notices/warnings going to the dashboard, can we just add an `if` conditional here and only send on `type == 1` ?

Reply:
I think only `type == 1` might result in under-reporting.. there are a number of different fatal error types in [the documentation](https://www.php.net/manual/en/errorfunc.constants.php).. some are compile errors, which I'm assuming refer not to the compilation of the php binary, but php's compilation of the PHP code.. I think syntax errors may be of this type.  I'm also noting `E_USER_ERROR` which says it's thrown if the php code calls `trigger_error()`.  Might want to check if WordPress and/or Drupal use that function.

Reply:
Then maybe filter on what we know we don't want to send? `type == 2 && type != 8`